fighters/trail/src/opff.rs

```diff
@@ -114,6 +114,15 @@ unsafe fn magic_cancels(boma: &mut BattleObjectModuleAccessor) {
             MotionModule::set_frame_sync_anim_cmd(boma, special_n_fire_cancel_frame_ground - landing_lag, true, true, true);
         }
     }
+    // Blizzard jump cancel
+    if (boma.is_status(*FIGHTER_TRAIL_STATUS_KIND_SPECIAL_N2)
+        && AttackModule::is_infliction_status(boma, *COLLISION_KIND_MASK_HIT) 
+        && VarModule::get_int(boma.object(), vars::common::instance::LAST_ATTACK_HITBOX_ID) == 0) {
+                boma.check_jump_cancel(false);
+                AttackModule::clear_all(boma);
+                WorkModule::off_flag(boma,  *FIGHTER_TRAIL_INSTANCE_WORK_ID_FLAG_MAGIC_SELECT_FORBID);
+                WorkModule::on_flag(boma,  *FIGHTER_TRAIL_STATUS_SPECIAL_N2_FLAG_CHANGE_MAGIC);
+    }
```

Comment:
please add a check to make sure the player is not currently in hitlag `!boma.is_in_hitlag()`

Reply:
also, `AttackModule::clear_all(boma);` is redundant: hitboxes will be cleared upon changing status

Reply:
Understood, have adjusted based off your feedback